src/core/Directus/Services/TablesService.php

```diff
@@ -102,7 +102,12 @@ public function findByIds($name, array $params = [])
 
         $tableGateway = $this->createTableGateway($this->collection);
 
-        return $tableGateway->getItemsByIds($name, $params);
+        $collection = $tableGateway->getItemsByIds($name, $params);
+        if(count($collection['data']) === 0){
```

Comment:
Find by Ids get into the the "filtering" category, which means if there's 0 result, is a valid response.